Directory.Packages.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="[1.0.0, 2.0)" />
+    <PackageVersion Include="System.Collections.Immutable" Version="1.7.1" />
+    <PackageVersion Include="System.Threading.Channels" Version="6.0.0" />
+  </ItemGroup>
```

Comment:
Might be worth adding a common here to explain that these are the runtime depencies where the list below is the testing deps

Reply:
How do you feel about just `Label="src"` and `Label="test"` to match the directories? Or were you thinking something more elaborate?

Reply:
yea thats fine i was just thinking using xml comments <!-- Runtime deps --> or something along those lines a label is fine as well